src/annotator/guest.js

```diff
@@ -79,11 +84,11 @@ export default class Guest extends Delegator {
     /** @type {ToolbarController|null} */
     this.toolbar = null;
 
-    this.adder = $(`<hypothesis-adder></hypothesis-adder>`)
-      .appendTo(this.element)
-      .hide();
+    this.adderToolbar = document.createElement('hypothesis-adder');
```

Comment:
Hooray! Can you remind me what the necessity is for using custom elements still? I've forgotten at this point.

Reply:
It helps to avoid generic selectors in the page's CSS interfering with the display of the element. Shadow DOM can prevent this more effectively for the content of Hypothesis's UI elements, but the shadow DOM still needs to be put in some kind of container which can be affected by the page's CSS.